dt/binding_secrets_test.go

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2018-2022 the original author or authors.
```

Comment:
```suggestion
 * Copyright 2018-2023 the original author or authors.
```

also needs updating in `binding_secrets.go`.

Reply:
Fixed in all changed go files.